pt-calculator-v2/src/main/java/org/egov/pt/calculator/web/models/Status.java

```diff
@@ -0,0 +1,38 @@
+package org.egov.pt.calculator.web.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * status of the Property
+ */
+public enum Status {
```

Comment:
Rename this to proper naming, like `BillStatus`, `PropertyStatus` depending on what this status is for